velox/docs/functions/presto/datetime.rst

```diff
@@ -6,6 +6,10 @@ Date and Time Functions
 
     Returns the current date.
 
+.. function:: current_timestamp() -> timestamp with time zone
```

Comment:
can you please also update the presto coverage map?
https://github.com/facebookincubator/velox/tree/main/velox/functions/prestosql/coverage